NewForce-at-Mountwest/celebrity-tribute-team-marky-mark
# Career page

I made a really great page with hover boxes and text.
How to test:
What you should do, is hover over the boxes and it should change from the movie poster to the image of the celebrity.
Hovering over the award boxes also makes them bigger.
Header should direct to my page and should highlight yellow when active.
Have a nice day
